test/test-ncu.js

```diff
@@ -95,14 +94,21 @@ describe('npm-check-updates', function () {
         });
 
         it('should fall back to package.json search when receiving empty content on stdin', function (done) {
-            var childProcess = child.exec('node bin/ncu', function (error, stdout) {
-                if (error) {
-                    done(error);
-                }
+            spawn('node', ['bin/ncu']).then(function (stdout) {
```

Comment:
I switched back to using done() from returning promises because you have more flow control when testing errors and promises. More of a concern when test driving an api that returns promises vs using promisey APIs in tests. I switched it back to returning to be consistent.